src/pages/api/thirdparty/operations/getPublicArticles.ts

```diff
@@ -0,0 +1,32 @@
+import { gql } from 'graphql-tag'
+
+export const GET_CNOTES = gql`
+  query GetCNotes($publishedDate: DateTime) {
+    articles(
+      where: {
+        status: { equals: Published }
+        publishedDate: { lte: $publishedDate }
+        category: { equals: InternalNews }
+        tags: { some: { name: { equals: "C-Note" } } }
+      }
+      orderBy: [{ publishedDate: desc }]
+    ) {
+      id
+      slug
```

Comment:
I noticed that `slug` isn't being returned in the api response, and I'm wondering if we need it?